data-sources/mux/config.yml

```diff
@@ -5,8 +5,10 @@ description: |
 icon: logo.svg
 install:
   primary:
-    link:
-      url: https://docs.newrelic.com/docs/logs/forward-logs/mux-video-streaming-firehose/
+    nerdletId: marketplace.install-data-source
```

Comment:
Looks like we need to add a field above nerdletId and nerdletState of `nerdlet`. Here's an example https://github.com/newrelic/newrelic-quickstarts/blob/370d7c9791f26c48e876e2dadb8b7d8ff4d229f5/data-sources/unix/config.yml#L10